README.md

```diff
@@ -134,6 +134,27 @@ lsw_memcache:
 
 Note that the session locking is enabled by default and the default spin lock is set to poll every 150 milliseconds (150000 microseconds).
 
+### Session Management for applications running behind a load balancer ###
+
+When your application is running on multiple servers you have to be aware that all your instances should be comunicating with 1 Caching server for consistency; otherwise each instance would have its own session and this would produce unexpected results.
+
```

Comment:
This can also be used for applications (different not load balanced) that share sessions.